src/Globals.cpp

```diff
@@ -195,6 +206,18 @@ namespace globals
 
 			ui = RE::UI::GetSingleton();
 			perFrame = { REL::RelocationID(524768, 411384) };
+
+			currentAccumulator = { REL::VariantID(527650, 414600, 0x3423058) };
```

Comment:
VR ID is released, just bump the vr address requirement to 0.182.0. In the end it doesn't matter.